### PR TITLE
scopegen: adding Any/All helpers for scopegen output_v2

### PR DESCRIPTION
**Pull request by favadi**

### CODE REVIEW

File: scopegen/generate/go.go [hunk 1]
```diff
@@ -65,6 +65,12 @@ scopes:
 	goTmplV2 = template.Must(template.New("").Parse(`// Code generated by "scopegen"; DO NOT EDIT.
 package {{ .Package }}
 
+type ServiceScope struct{}
+
+func NewServiceScope() *ServiceScope {
```
Comment (Kunde21) [inline scopegen/generate/go.go]: We should need a constructor.  Embedding the type in the service struct definition will take care of it.

Comment (favadi) [reply scopegen/generate/go.go]: > We should need a constructor.

You meant we don't need the constructor?

> Embedding the type in the service struct definition will take care of it.

Already did that in the other PR.

Comment (favadi) [reply scopegen/generate/go.go]: I removed the constructor in latest commit, as we used it nowhere.

Comment (Kunde21) [reply scopegen/generate/go.go]: You're right, that's what I meant.  Thank you.

**Pull request merged**